zebrad/src/components/mempool/error.rs

```diff
@@ -23,7 +23,7 @@ pub enum MempoolError {
     ///
     /// Note that the mempool caches this error. See [`super::storage::Storage`]
     /// for more details.
-    #[error("mempool storage has a cached tip rejection for this exact transaction")]
+    #[error("the authorizing data of the transaction conflicts with the current chain tip")]
```

Comment:
I want to say something like "this exact transaction" or "identical transactions" here.
What do you think?

```suggestion
    #[error("the transaction will be rejected from the mempool until the next chain tip block")]
```